src/services/signalr.ts

```diff
@@ -25,13 +25,15 @@ export class SignalR {
             let configuration = this.merge(options ? options : {});
 
             try {
-                let serialized = JSON.stringify(configuration.qs);
+                let serializedQs = JSON.stringify(configuration.qs);
+                let serializedTransport = JSON.stringify(configuration.transport);
 
                 if (configuration.logging) {
                     console.log(`Connecting with...`);
                     console.log(`configuration:[url: '${configuration.url}'] ...`);
                     console.log(`configuration:[hubName: '${configuration.hubName}'] ...`);
-                    console.log(`configuration:[qs: '${serialized}'] ...`);
+                    console.log(`configuration:[qs: '${serializedQs}'] ...`);
+                    console.log(`configuration:[transport: '${serializedTransport}'] ...`);
```

Comment:
you can change this to console.log(`configuration:[transport: '${configuration.transport}'] ...`); ?